### ISSUE TITLE
oomph setup not working

### ISSUE THREAD

**Issue by Bananeweizen** [m2e-code-quality/m2e-code-quality#201]
Inspired by your blog post, I ran the oomph setup on my personal Windows machine. It looks fine until the IDE starts. There is zero activity (normally git cloning would start), and invoking "Perform setup tasks" leads to an empty list of setup tasks. Also opening all the Navigate -> Setup -> xyz leads to empty setup files (except for my user setup, so generally, Oomph is working).

Since I have not seen such a behaviour with any of the many Oomph setups I manage myself, I'm slightly confused what to investigate. Might this be an issue with the oomph setup redirection? Do you have any other suggestion what to investigate locally?

**Comment by vorburger** [comment]
> Inspired by your blog post

happy to hear that it inspires... :smiley: 

> I'm slightly confused what to investigate

Dunno - you COULD debug Oomph itself - but not sure it's "worth" it?

May be others here want to chime in and have a closer look... I won't have time.

**Comment by vorburger** [comment]
BTW is #197 a dupe of this? Thought perhaps you should know about that issue.

**Comment by Bananeweizen** [comment]
No. That other issue is about resolving the target platform. Errors there are typically caused by
* local oomph cache pointing to a nightly update site version, which was replaced during this nights build
* modular targets becoming incompatible due to someone upgrading one modular part, and not checking the others

I'm fine with closing this issue, since I've cloned the repo myself. I'm just not sure, if you set any workspace preferences in the oomph setup (I hope you did not), which would be missing in my case.
I've also just noticed that Oomph itself got updated today, and of course I used that new release for installing. I will try to find an old version of the installer and try again without updating the installer itself.
